docs/README.md

```diff
@@ -23,12 +23,103 @@ Marketplace features included in Saunatime:
 - **User profiles** provide detailed information about a given user.
 - **Extended data:** The listing and user data models are modified using extended data.
 
-## Flex Docs website
-
-We are currently moving documentation to the Flex Docs site, so at the moment part of the
-documentation is in this repository and part is in the Flex Docs site.
-
-See the Flex Docs site: https://www.sharetribe.com/docs/
+## Directory structure
+
+After cloning the repo, your project should look like this:
+
+```
+├── ext
+│   └── default-mail-templates
+├── package.json
+├── docs
+├── public
+│   ├── 500.html
+│   ├── index.html
+│   ├── robots.txt
+│   └── static
+├── server
+│   ├── index.js
+│   ├── csp.js
+│   ├── sitemap.js
+│   └   ...
+└── src
+    ├── Routes.js
+    ├── analytics
+    ├── app.js
+    ├── assets
+    ├── components
+    ├── config.js
+    ├── containers
+    ├── ducks
+    ├── examples.js
+    ├── forms
+    ├── index.js
+    ├── marketplace-custom-config.js
+    ├── marketplace.css
+    ├── marketplaceFonts.css
+    ├── marketplaceIndex.css
+    ├── reducers.js
+    ├── routeConfiguration.js
+    ├── store.js
+    ├── translations
+    └── util
```

Comment:
A small suggestion, this might be more understandable if directories would include a trailing slash `util/`.

Reply:
I just copied the content from the existing doc, but you are right. I can change it here.

Reply:
Fixed.